src/database/world_graph.lua

```diff
@@ -359,10 +364,11 @@ function plantation:new(worldGraph)
     if not self.world.regions.arthur:canAccess(items) then return false end
     if _has(items, "teleportKey") then return true end
     if self.world.regions.outerWall:canAccess(items) then return true end
+    if _has(items, "bomb") and _has(items, "weaponSN") and self.world.regions.grasstownEast:canAccess(items) then return true end
     return false
   end
 
-  self.locations.jail1.requirements = function(self, items) return _has(items, "letter") end
+  self.locations.jail1.requirements = function(self, items) return _has(items, "letter") and _has(items, "teleportKey") end
```

Comment:
the `letter` requirement actually shouldn't be here at all lol I must've mixed up the requirement with the vanilla item